includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -881,6 +881,11 @@ private function update_woopay_enabled_locations( WP_REST_Request $request ) {
 			return;
 		}
 
+		$is_woopay_enabled = 'yes' === $this->wcpay_gateway->get_option( 'platform_checkout', 'no' );
```

Comment:
Can we reuse the [`is_woopay_enabled` function](https://github.com/Automattic/woocommerce-payments/blob/6ea513ef9ec64812aa214af8779b376499b74b22/includes/class-wc-payments-features.php#L125) here?

Reply:
ah thanks! I forgot about that function. Updated in https://github.com/Automattic/woocommerce-payments/pull/7244/commits/7250cb687f242797fd93e223877e749bff6c5fe7